### DISCUSSION TITLE
Cypress handle child window

### DISCUSSION THREAD

**Discussion by pradhp05** [cypress-io/cypress#23895] [Questions and Help]
I come across a situation where I cant able to access the child windows in cypress, like if I link on a button it opens up a new child windows to login so I cant able to access that child windows.  Needed help on it  

Thanks in advance!!!.

**Comment by jjhelguero** [comment]
There are several scenarios how your app can open a new window. This may help you: https://glebbahmutov.com/blog/cypress-second-tab/.
